aws/scenarios/vm/ec2VM/ec2VM.go

```diff
@@ -20,14 +24,22 @@ func NewEc2VM(ctx *pulumi.Context, options ...func(*Params) error) (vm.VM, error
 	}
 
 	os := params.common.OS
+	userData := params.common.UserData
+	if os.GetType() == commonos.WindowsType {
+		cmd, err := GetOpenSSHInstallCmd(env.DefaultPublicKeyPath())
+		if err != nil {
+			return nil, err
+		}
+		userData += cmd
```

Comment:
I'm not sure that works. Maybe the added script should be inside `<powershell>`

Reply:
It works at least based on my test (You can have `<powershell> ... </powershell><powershell> ... </powershell>`